am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestDataFactory.java

```diff
@@ -52,6 +62,12 @@ public static ExplicitAccessGrant createGrant(String resourceId,
             .build();
     }
 
+    public static ExplicitAccessGrant createGrant(String resourceId,
```

Comment:
Move above `createGrant(String resourceId, Set<String> accessorId, Map<JsonPointer, Set<Permission>> attributePermissions)` so it reads more naturally.

Reply:
👍